ganttproject/src/net/sourceforge/ganttproject/gui/UIFacade.java

```diff
@@ -154,4 +155,6 @@
   GPOptionGroup[] getOptions();
 
   void addOnUpdateComponentTreeUi(Runnable callback);
+
+  void showUpdateDialog(RssUpdate update);
```

Comment:
I think that this UIFacade layer is not necessary. You can just call `UpdateDialog` from `RssFeedChecker`